administrator/components/com_media/resources/scripts/components/browser/items/directory.vue

```diff
@@ -9,8 +9,20 @@
       @dblclick.stop.prevent="onPreviewDblClick()"
       @keyup.enter="onPreviewDblClick()"
     >
-      <div class="file-background">
-        <div class="folder-icon">
+      <div
+        class="file-background"
+        :class="{ 'with-thumbnail': thumbURL }"
+      >
+        <img
+          v-if="thumbURL"
+          class="image-cropped"
+          alt=""
+          :src="thumbURL"
+          :loading="thumbLoading"
+          :width="thumbWidth"
+          :height="thumbHeight"
+        >
+        <div v-if="!thumbURL" class="folder-icon">
```

Comment:
```suggestion
        <div v-if="!thumbURL"
          class="folder-icon"
        >
```
Fix javascript-cs (2 of 5)